site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -88,15 +88,31 @@ const CourseTile: FC<CourseTileProps> = ({ courseID, dragTimestamp = 0, taken })
     fontSize = computedSize + 'px';
   }
 
+  const sourceOverlay =
+    completedBy === 'AP' ? (
+      <p className="source-overlay" title={`${courseID} is cleared by an AP exam`}>
+        AP
+      </p>
+    ) : completedBy === 'Course' ? (
+      <p className="source-overlay" title={`${courseID} is cleared by a transferred course`}>
+        Course {/* to do: replace with MUI icon */}
+      </p>
+    ) : null;
+
```

Comment:
Consider putting this into its own component since the logic is more involved than I'd like for inline.

that component would looks something like:
```tsx
const SourceOverlay: FC<...> = ({ completedBy: ... }) => {
  if (!completedBy || completedBy === 'roadmap') return null;
  const title = 'Cleared by ' + (completedBy === 'AP' ? 'an AP Exam' : 'a transferred course')
  const icon = completedBy === 'AP' ? 'AP' : <Icon/>
  return <p className="source-overlay" title={title}>{icon}</p>
}
```